### DISCUSSION TITLE
Some Photos dont show up in mobile app.

### DISCUSSION THREAD

**Discussion by carloslockward** [immich-app/immich#14941] [Q&A] [answered]
### The bug

I just noticed some pictures I took were missing in the Immich mobile app on Android. The photos themselves don't seem any different from other pictures I've taken.

I tried restarting, force stopping, and rebooting the phone, but those pictures just don't show up.

They show up fine in the default gallery and Google Photos.

Assets are all in the same folder on my phone (/storage/emulated/0/DCIM/Camera) . They are the same format and appear to have similar metadata. (Some were taken mere moments apart and one shows up and the other doesn't)

Its very strange.

![Screenshot_20241226_021243](https://github.com/user-attachments/assets/7e7f8a45-b095-4aff-bfe5-711dce6734f1)
![Screenshot_20241226_021459_Photos](https://github.com/user-attachments/assets/5af2e516-8fa0-4e92-8c6e-7e6358565112)

### The OS that Immich Server is running on

unRAID

### Version of Immich Server

v1.123.0

### Version of Immich Mobile App

v1.123.0 build.172

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

Unknown so far
...

**Comment by carloslockward** [comment]
@bo0tzz Why dismiss this so quickly as a "discussion" and not a bug? Also, if you know why this is happening, let me know.

**Comment by carloslockward** [comment]
@alextran1502 Any thoughts on this? haven't been able to find a solution

**Reply by alextran1502** [reply, marked as answer]
Can you try reinstall the app?

**Reply by carloslockward** [reply]
Just did. That did fix it (after starting the backup again.) Thanks for the help. Wonder what caused this in the first place tho.

**Reply by alextran1502** [reply]
I think the post-processing in the Photos app altered the file's content (hash), causing the hiccup
